docs/docs/examples/guides/camera-calibration.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 9
```

Comment:
On main, position 9 is technically already assigned. We need better ordering anyway but for now I would say you either have to shift the robot guides backwards or you just put this at the end

Reply:
hm yes, I didn't check again after the latest PR merge, thanks. I'll put those at the end, I think (for now) it's enough to have the 3 upcoming guides back-to-back